config/routes.rb

```diff
@@ -107,6 +107,11 @@
             end
           end
         end
+        namespace :tooltips do
+          resource :tooltip, controller: "tooltip", only: [:show] do
+            get 'student/:id', to: 'tooltip#student', as: 'student'
```

Comment:
I'd personally do

```ruby
resources :students, only: %i[] do
  resource :tooltip, only: %i[show], controller: 'tooltip', action: 'student'
end
```

or


```ruby
resource :tooltip, only: %i[show], controller: 'tooltip' do
  resources :students, only: %i[show], controller: 'tooltip/students'
end
```

But I know Jeremy likes to be non-rails about all this.

Reply:
@ErikSchierboom You should entirely ignore DJ's comment here 😉

Reply:
I'd like to preserve my objection for the record 💯